README.md

```diff
@@ -26,8 +26,41 @@
 
 ## Usage
 
+Check if an arbitrary process seems to be running (including more than one instance or not) or not running. Our arbitrary process in this example is called `rotgut`. Usage of `check-process.rb` would look something similar to the following:
+
+    $ /opt/sensu/embedded/bin/ruby /opt/sensu/embedded/bin/check-process.rb -p gutrot
+    CheckProcess OK: Found 3 matching processes; cmd /gutrot/
+
+The `-p` argument is for a patter to match against the list of running processes
+reported by `ps`.
+
+Example configuration at `/etc/sensu/conf.d/check_gutrot_running.sh`:
+
+    {
+      "checks": {
+        "check_gutrot_running": {
+          "type": "metric",
+          "command": "/opt/sensu/embedded/bin/ruby /opt/sensu/embedded/bin/check-process.rb -p gutrot",
```

Comment:
with embedded ruby being in the path by default you only need to run `check-process.rb` and not the full path to the ruby and the check.